### ISSUE TITLE
Rebel recruiting generates allLaserAttachments error at night

### ISSUE THREAD

**Issue by jaj22** [official-antistasi-community/A3-Antistasi#510]
Version: Unstable.

When you recruit either rebel squads or units at night, you get this script error locally:

``` 3:10:33 Error in expression <dom unlockedNVGs)};
private _pointers = allLaserAttachments arrayIntersect unloc>
 3:10:33   Error position: <allLaserAttachments arrayIntersect unloc>
 3:10:33   Error Undefined variable in expression: alllaserattachments
 3:10:33 File functions\REINF\fn_FIAinit.sqf [A3A_fnc_FIAinit]..., line 108
```

Probably because allLaserAttachments isn't being made public, and FIAinit is called on the commander's client.

While this is an easy fix, I'd rather rebels didn't use the lasers, personally. It doesn't make much sense given that their enemies have NV, and without the lasers it's a lot easier to avoid friendly fire at night.

**Comment by jaj22** [comment]
Fixed this in #583. Can be closed.

**Comment by Bob-Murphy** [comment]
Thank you.
